react/features/stream-effects/screenshot-capture/ScreenshotCaptureEffect.js

```diff
@@ -57,11 +57,7 @@ export default class ScreenshotCaptureEffect {
      * @returns {boolean} - Returns true if this effect can run on the specified track, false otherwise.
      */
     isEnabled(jitsiLocalTrack: Object) {
```

Comment:
I can't find where is this method used?

Reply:
This is used in LJM setEffect when we try to apply the effect on the JitsiLocalTrack.

Reply:
okay.. eventually I've found it's in LJM